pkg/cmd/kafka/update/update.go

```diff
@@ -125,6 +127,7 @@ func NewUpdateCommand(f *factory.Factory) *cobra.Command {
 	return cmd
 }
 
+// nolint:funlen
```

Comment:
Ideally this should be removed before we merge.

Reply:
Lets remove that

Reply:
We need it as the run method fails with `Function 'run' has too many statements (58 > 50)`

Reply:
Can we make function have less statements :)

Reply:
Not addressed

Reply:
The method has 120+ lines of code, same goes for multiple places where we have excluded lints.

Reply:
Not sure what you mean. Do you mean that we do not want to address that line? or suggestion is to increase limit?

Reply:
I think this should be addressed in a separate issue, funlen check is skipped at several places and the the lines in this method is more than twice the limit.

Reply:
I fully understyand what you are saying but that is new place introduced in that specific PR. Using that logic every of review comment can be addressed as separate github issue. I'm really giving up on this.